modules/web/index.html

```diff
@@ -28,6 +28,9 @@
 
     <script src="lib/css-browser-selector_0.4.0/css_browser_selector.js"></script>
 
+    <!-- loading monospaced font for source view -->
+    <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
```

Comment:
The font uses OFL(Source Code Pro can be adapted and redistributed according to the terms of the Open Font License (OFL) agreement.[1]) license. @hevayo should we be concerned ?

[1] - https://fonts.google.com/specimen/Source+Code+Pro